### ISSUE TITLE
训练好的模型在MoCA-Mask测试集上的表现不好

### ISSUE THREAD

**Issue by xixioo123** [lartpang/ZoomNeXt#15]
作者您好，我按照您的步骤进行预训练和微调，训练好的模型在MoCA-Mask测试集上的表现不好（num_frames = 5）
![image](https://github.com/user-attachments/assets/3ef8cf6f-cf92-4c53-b7e5-994729d07c42)
预训练使用 3040张来自COD10K   和1000张来自CAMO
预训练配置文件icod_pretrain更改了BATCHSIZE为8以及添加了训练数据camo_tr       names=["cod10k_tr", "camo_tr"],：
vcod_finetune配置文件 没有更改
评估使用的是您提供的工具PySODEvalToolkit    
我应该是哪里出现问题了呢？ 
另外评估代码中    --valid-frame-start "0" --valid-frame-end "-2" 在不同的T值的情况下要不要更改？
期待您的回复。

**Comment by xixioo123** [comment]
我的数据集路径设置：moca_mask_tr:
  {
    root: "datasets/MoCA-Mask/MoCA_Video/TrainDataset_per_sq",
    image: { path: "*/Imgs", suffix: ".jpg" },
    mask: { path: "*/GT", suffix: ".png" },
  }
moca_mask_te:
  {
    root: "datasets/MoCA-Mask/MoCA_Video/TestDataset_per_sq",
    image: { path: "*/Imgs", suffix: ".jpg" },
    mask: { path: "*/GT", suffix: ".png" },
  }
cod10k_tr:
  {
    root: "datasets/COD10K3040/COD10K-v2/Train",
    image: { path: "Image", suffix: ".jpg" },
    mask: { path: "GT_Object", suffix: ".png" },
  }
cod10k_te:
  {
    root: "datasets/COD10K3040/COD10K-v2/Test",
    image: { path: "Image", suffix: ".jpg" },
    mask: { path: "GT_Object", suffix: ".png" },
  }

camo_tr:
  {
    root: "datasets/CAMO-V.1.0-CVIU2019/Train",
    image: { path: "Img", suffix: ".jpg" },
    mask: { path: "GT", suffix: ".png" },
  }
camo_te:
  {
    root: "datasets/CAMO-V.1.0-CVIU2019/Test",
    image: { path: "Img", suffix: ".jpg" },
    mask: { path: "GT", suffix: ".png" },
  }
cad:
  {
    root: "datasets/CamouflagedAnimalDataset",
    image: { path: "*/frames", suffix: ".png" },
    mask: { path: "*/groundtruth", suffix: ".png" },
  }

**Comment by lartpang** [comment]
@xixioo123 

- On one hand, during the pretraining phase, we remain consistent with SLT-Net by using only the COD10K dataset for pretraining.  
- On the other hand, we adopt SLT-Net's "pretraining + fine-tuning" design. This cascaded learning process can, to some extent, impact the final performance of the video model.  
- During testing, the method listed in the README ensures consistency with SLT-Net. It does not require modification for different training configurations.

**Comment by xixioo123** [comment]
非常感谢您的回答
